### DISCUSSION TITLE
Changing colour of light?

### DISCUSSION THREAD

**Discussion by dlipner** [petretiandrea/home-assistant-tapo-p100#295] [Q&A]
Is it possible to use this to set up an automation to change the colour of the bulb, rather than just turn it on or off?

**Comment by petretiandrea** [comment]
Yes, you can use home assistant automation. But this is not the right place to ask for, please refers to home assistant documentation

**Reply by dlipner** [reply]
I know how to use the automations and write script, but what I can’t find is the code/command to change the light colour and brightness. Only on / off / toggle
